core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -112,7 +115,20 @@ class SocketServer(val config: KafkaConfig, val metrics: Metrics, val time: Time
         }.sum / totalProcessorThreads
       }
     )
-
+    val memoryPoolDepletedPercentMetricName = metrics.metricName("MemoryPoolAvgDepletedPercent-Avg", "socket-server-metrics")
+    val memoryPoolSensor = metrics.sensor("MemoryPoolAvgDepletedPercent")
+    memoryPoolSensor.add(memoryPoolDepletedPercentMetricName, new Rate(TimeUnit.MILLISECONDS))
```

Comment:
Since this is a server side metric, it's probably better to use a Yammer metric to be consistent. Currently, we try only using the client metric on the server side if it needs additional functionality from the client metric (e.g., quota).